src/Airac.cpp

```diff
@@ -107,69 +107,41 @@ void Airac::readAirways(const QString& directory) {
             continue;
 
```

Comment:
Let's add a comment with what a line looks like.
```suggestion
// ABDOR GM 11 VALBA GM 11 N 2 195 460 UZ801
// https://developer.x-plane.com/article/navdata-in-x-plane-11/
```